package.json

```diff
@@ -23,7 +23,7 @@
     "test": "grunt test"
   },
   "dependencies": {
-    "loopback-sdk-angular": "^1.2.0"
+    "loopback-sdk-angular": "https://github.com/aquid/loopback-sdk-angular/tarball/master"
```

Comment:
Even better, change the dependency version spec to `^1.10.0` to ensure we have a version that supports `modelsToIgnore` option.